src/js/get_profile_data.js

```diff
@@ -45,6 +45,10 @@
 
     const defaultAliasLabelText = browser.i18n.getMessage("profilePageDefaulAliasLabelText");
     const storedAliasLabel = (addonRelayAddress && addonRelayAddress.hasOwnProperty("domain")) ? addonRelayAddress.domain : "";
+    
+    // Cache the siteOrigin alias attribute when updating local storage data.
+    // Note that this data attribute only exists in aliases generated through the add-on
+    const storedAliasSiteOrigin = (addonRelayAddress && addonRelayAddress.hasOwnProperty("siteOrigin")) ? addonRelayAddress.siteOrigin : "";
```

Comment:
As this data only exists from the add-on/local storage, it has to be queried the same was as the alias label is.

Reply:
Some small syntactic sugar you might be interested in :)

```suggestion
    
    // Cache the siteOrigin alias attribute when updating local storage data.
    // Note that this data attribute only exists in aliases generated through the add-on
    const storedAliasSiteOrigin = addonRelayAddress?.siteOrigin ?? "";
```